cli/command/service/list.go

```diff
@@ -40,6 +36,8 @@ func newListCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Only display IDs")
+	flags.BoolVar(&opts.noTrunc, "no-trunc", false, "Do not truncate the output")
```

Comment:
@thaJeztah does that mean we will leave `ID` alone regardless of `--no-trunc=false/true`, and only trunc `TruncID` based on `--no-trunc=true`?